src/pages/VEaas.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Fragment } from 'react';
+import { Button, Card, CardBody, Col, Row } from 'reactstrap';
+import ReactPlayer from 'react-player';
+import { useRouteData } from 'react-static';
+import { Link } from '@reach/router';
+import sk from '../assets/images/Sk.jpg';
+import dream from '../assets/images/dream.jpg';
+
+const VE = () => {
```

Comment:
```suggestion
const VEaaS = () => {
```
Component names should match file name